src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -226,15 +227,14 @@ const styles = {
   }),
 };
 
-const parseBucketValue = value => (value % 1
-  ? Number.parseFloat(value).toFixed(2)
+const parseContinuousBucketValue = value => (value % 1
+  ? value
   : Math.round(value * 100) / 100);
 
-const getRangeValue = (key, nextInterval) => `${parseBucketValue(key)}${
-  nextInterval === 0
-    ? ' and up'
-    : ` to ${parseBucketValue(nextInterval - 1)}`
-  }`;
+const getContinuousRangeValue = keyValues => {
```

Comment:
change how values are parsed - integers come back from the API with a `.0` suffix

use integers or up to 2 decimal places, no more `.0`